examples/map/masking_hmi.py

```diff
@@ -0,0 +1,73 @@
+"""
+=========================================
+Masking HMI based on the intensity of AIA
+=========================================
+
+In this example we will demonstrate how to mask out regions within a
+HMI map based on the intensity values of AIA.
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+from skimage.measure import label, regionprops
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+
+import sunpy.map
+from sunpy.data.sample import AIA_171_IMAGE, HMI_LOS_IMAGE
+
+###############################################################################
+# We will use an AIA 171 image from the sample data and crop it to capture a region of interest.
+
+aia = sunpy.map.Map(AIA_171_IMAGE)
+aia = aia.submap(
+    bottom_left=SkyCoord(-250, 200, unit=u.arcsec, frame=aia.coordinate_frame),
+    width=500 * u.arcsec,
+    height=400 * u.arcsec,
+)
+hmi = sunpy.map.Map(HMI_LOS_IMAGE)
+
+###############################################################################
+# Next we call the :meth:`~sunpy.map.GenericMap.reproject_to` to reproject the HMI Map
+# to have exactly the same grid as the AIA Map.
+# :ref:`sphx_glr_generated_gallery_map_transformations_reprojection_align_aia_hmi.py` provides more reference.
+
+hmi = hmi.reproject_to(aia.wcs)
+
+###############################################################################
+# Now we will identify separate regions below a threshold in the AIA Map.
+# In this case, we want the darker patches that have counts below 200.
```

Comment:
```suggestion
# In this case, we want the darker patches that have pixel values below 200.
```
Despite the fact that the AIA map units are reported as "counts", they are actually DN.  To avoid that all, just say "pixel values".